qiime2/sdk/plugin_manager.py

```diff
@@ -100,11 +101,28 @@ def _integrate_plugin(self, plugin):
 
             self.formats[name] = record
         self.type_formats.extend(plugin.type_formats)
+        self.importable_types.update(plugin.importable_types)
```

Comment:
Don't forget to update to the new datatype. Let's use the `plugin.name` value as the key (although I am open to other ideas, like using the plugin object outright, instead).

```suggestion
        self.types[plugin.name] = plugin.types
```